src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    String getPrintableString();
+    String toString();
+    boolean equals(Object o);
```

Comment:
These methods (toString, equals, hashCode) shouldn't be ins your interface, as they aren't related to your Printable interface, and are _overriden_ methods from the default toString/equals/hashCode methods in Java.